esrally/mechanic/telemetry.py

```diff
@@ -431,14 +431,17 @@ def record_stats_per_index(self, name, stats):
 
         for shard_stats in stats:
             if "shard_id" in shard_stats:
+                doc = {
+                    "name": "ccr-stats",
+                    "shard": shard_stats
+                }
                 shard_metadata = {
                     "cluster": self.cluster_name,
                     "index": name,
-                    "shard": shard_stats["shard_id"],
-                    "name": "ccr-stats"
+                    "shard": shard_stats["shard_id"]
```

Comment:
This is included in the shard stats already. Does it make sense to duplicate this information?

Reply:
Whereas here I think it doesn't make sense to duplicate the shard_id; I'll remove.